smart-contracts/faucets.md

```diff
@@ -1 +1,44 @@
-TODO
\ No newline at end of file
+---
+title: Faucets
+description: Learn how to obtain test tokens from Polkadot faucets for development and testing purposes across different networks.
+categories: Smart Contracts
+---
+
+# Faucets
+
+Test tokens allow you to experiment with smart contracts, test transactions, and validate your dApp functionality without using real cryptocurrency. Polkadot has a reliable faucet that distributes free TestNet tokens to developers for testing purposes.
+
+## Faucet Basics
+
+Faucets are services that distribute small amounts of cryptocurrency or tokens for free, typically on TestNet networks. They serve as the primary method for developers to obtain the necessary tokens to:
+
+- Execute transactions on TestNet networks.
+- Pay for gas fees when deploying smart contracts.
+- Test dApp functionality before MainNet deployment.
+- Experiment with blockchain interactions without financial risk.
+
+## Get Test Tokens
+
+For Polkadot Hub TestNet, you can use the [Polkadot Faucet](https://faucet.polkadot.io/?parachain=1111){target=\_blank} to obtain test tokens. Here's how to do it:
+
+1. Navigate to the [Polkadot Faucet](https://faucet.polkadot.io/?parachain=1111){target=\_blank}. If the desired network is not already selected, choose it from the Network drop-down. This example uses the Polkadot Hub TestNet.
```

Comment:
Is Network a UI element? If so, it should be in bold:

> If the desired network is not already selected, choose it from the **Network** drop-down.

The last sentence here says "This example uses the Polkadot Hub TestNet." but in the intro sentence we already state that. So it's redundant, I would remove it.